projects/valtimo/resource/src/lib/services/open-zaak.service.ts

```diff
@@ -56,6 +56,10 @@ export class OpenZaakService {
     return this.http.get<ZaakType[]>(`${this.valtimoApiConfig.endpointUri}openzaak/zaaktype`);
   }
 
+  getBesluitTypes(): Observable<any> {
+    return this.http.get(`${this.valtimoApiConfig.endpointUri}besluit/types`);
```

Comment:
Be careful with this. might need to change if the backend does

Reply:
Changed it